extensions/wikia/AdEngine/js/template/skin.js

```diff
@@ -7,8 +7,7 @@ define('ext.wikia.adEngine.template.skin', [
 ], function (adContext, doc, win, log) {
 	'use strict';
 
-	var logGroup = 'ext.wikia.adengine.template.skin',
-		sevenOneMedia = adContext.getContext().providers.sevenOneMedia;
```

Comment:
If I read the code correct the `sevenOneMedia` variable wasn't used anyway :stuck_out_tongue_winking_eye: